packages/members/layer2/index.js

```diff
@@ -8,10 +8,11 @@ function hide (el) {
 }
 
 module.exports = {
-    init: function init() {
+    init: function init(options) {
         var members = Members.create();
 
         var signin = document.querySelector('[data-members-signin]');
+        var signinCta = document.querySelector('[data-members-signin-cta]');
```

Comment:
Rather than adding an extra selector/data-sstribute for each new piece of functionality, should we instead use `querySelectorAll` and support as many as the user likes? seems more flexible maybe!

Reply:
Yeah i actually started by changing it to `querySelectorAll` and updating code, but soon realized cta behavior is different from header button as it needs to handle `login` part but not `render` part because link is part of whole CTA text. So reverted to this for now, but agree we'll def need to clean it up better.

Reply:
Hmmm, I don't think its a problem for the cta to be hidden on login? But yeah this deffo needs cleaning up